web/src/components/Socials.tsx

```diff
@@ -0,0 +1,34 @@
+import { FaFacebookF, FaInstagram, FaLinkedin } from "react-icons/fa";
+import { IoMdMail } from "react-icons/io";
+
+export default function Socials() {
+	return (
+		<>
+			<div className="w-60 flex justify-between items-center py-2" data-testid="socialsComponent">
+				<div className="w-10 h-10 bg-primary-orange rounded-full hover:bg-light-pink cursor-pointer group">
+					<a href="https://www.instagram.com/au.indiansociety/?hl=en" target="_blank" rel="noopener noreferrer" aria-label="Instagram" className="w-full h-full flex justify-center items-center">
```

Comment:
Are you able to put all the social media links in the data file? Not a good idea to hard code values in the component as it can be difficult to change